RELEASE_NOTE.md

```diff
@@ -5,7 +5,14 @@
 - Marketplace related update
   - Please see [Marketplace](https://github.com/microsoft/openpaimarketplace/releases/tag/v1.7.0) for more details
 
-- Introduce new job submission page
+- New job submission page
+  - Please refer to [new submission tutorial] for how to use new submission page.
```

Comment:
[new submission tutorial] add link for this?

Reply:
currently the tutorial is in a pull request and will merged later today. So the link might not work right now